src/Compilers/Core/Portable/CodeGen/ArrayMembers.cs

```diff
@@ -362,6 +362,20 @@ public void Dispatch(Cci.MetadataVisitor visitor)
             => null;
 
         public override string ToString()
-            => arrayType.ToString() + "." + Name;
+            => ((object?)arrayType.GetInternalSymbol() ?? arrayType).ToString() + "." + Name;
+
+        Symbols.ISymbolInternal? Cci.IReference.GetInternalSymbol() => null;
+
+        public sealed override bool Equals(object? obj)
+        {
+            // It is not supported to rely on default equality of these CCi objects, an explicit way to compare and hash them should be used.
```

Comment:
>CCi [](start = 72, length = 3)

"CCI" (uppercase "I")?

Reply:
> "CCI" (uppercase "I")?

It actually should be "Cci", that is how the namespace named.

---
In reply to: [512321416](https://github.com/dotnet/roslyn/pull/48838#discussion_r512321416) [](ancestors = 512321416)